test/wdio/host-attr-override/host-attr-override.test.tsx

```diff
@@ -0,0 +1,23 @@
+import { h, Fragment } from '@stencil/core';
+import { render } from '@wdio/browser-runner/stencil';
+
+describe('host attribute overrides', function () {
+  beforeEach(() => {
+    render({
+      template: () => (
+        <>
+          <host-attr-override class="override"></host-attr-override>
+          <host-attr-override class="with-role" role="another-role"></host-attr-override>
+        </>
+      ),
+    });
+  });
+
+  it('should merge class set in HTML with that on the Host', async () => {
+    expect($('.default.override')).toExist();
```

Comment:
```suggestion
    await expect($('.default.override')).toExist();
```

Reply:
Unless this auto-retries? 🤔

Reply:
nope, it can't without `await`, good catch 👍